### PR TITLE
Added v2 end points

### PR DESCRIPTION
**Pull request by defong**
### Context
v2 end points

### Changes proposed in this pull request
Added v2 end points

### CODE REVIEW

**Comment by defong** [comment]
[Create api/v2 for manage courses frontend](https://trello.com/c/ChXP2RQm/950-create-api-v2-for-manage-courses-frontend)

File: config/routes.rb [hunk 1]
```diff
@@ -30,6 +54,12 @@
         resources :courses
       end
     end
+    namespace :v2 do
+      resources :providers
+      resources :subjects
+      resources :courses
+      resources :organisations
+    end
```
Comment (misaka) [inline config/routes.rb]: These don't have controllers (yet). That means if someone hits these endpoints they get a 500 not a 404. I think we should only add these routes in when we have a controller to respond to it.

Comment (defong) [reply config/routes.rb]: I was getting distracted with this...
https://github.com/rails-api/active_model_serializers/issues/144

regards to 404 vs 500

image

Comment (misaka) [reply config/routes.rb]: Ah, good spot. I still wonder about adding the routes first without having controllers for them, but at least it doesn't 500 then.

**Comment by defong** [comment]
refocusing on depth vs breadth

**Pull request closed without merging**